package.json

```diff
@@ -39,7 +39,7 @@
       "browser": true
     },
     "parserOptions": {
-      "ecmaVersion": 6,
+      "ecmaVersion": 2018,
```

Comment:
To support `\p{L}` syntax in the regexp.  Supported by all major browsers except IE/Edge